Admin/AbstractAdmin.php

```diff
@@ -577,14 +565,6 @@ public function getDataSourceIterator()
         foreach ($this->getExportFields() as $key => $field) {
             $label = $this->getTranslationLabel($field, 'export', 'label');
             $transLabel = $this->trans($label);
-
-            // NEXT_MAJOR: Remove this hack, because all field labels will be translated with the major release
-            // No translation key exists
-            if ($transLabel == $label) {
-                $fields[$key] = $field;
-            } else {
-                $fields[$transLabel] = $field;
-            }
```

Comment:
This does not look right, `$transLabel` is unused. I think line 589 should be kept.